test/spec/asyncVersioningSpec.js

```diff
@@ -2,7 +2,7 @@ if(typeof JSONPatchQueue === 'undefined') {
   JSONPatchQueue = require('../../src/index').JSONPatchQueue;
 }
 
-var obj;
+var obj = {foo: 1, baz: [{qux: 'hello'}]};
```

Comment:
The "global" is also not so good, as this is the same object reference shared between all instances - it makes them not so independent. I'd recreate it for each.